converter/converter.py

```diff
@@ -97,6 +98,13 @@ def request(
     return openapi
 
 
+def styled_error(error: str, p: Path) -> str:
```

Comment:
I'd prefer if functions would use proper names, like `path` for arguments, rather than short one letter variables like `p`.